ct/paperless-ngx.sh

```diff
@@ -114,6 +133,11 @@ function update_script() {
       cd /opt/paperless/src
       $STD uv run -- python manage.py migrate
       msg_ok "Paperless-ngx migration and update completed"
+
+      if [[ -d /opt/paperless/backup ]]; then
```

Comment:
Why do you test for existence of the backup dir since you created it with mkdir -p before. It will be there.